app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -44,4 +44,44 @@ public function cleanupQuotes() {
 
         $connection->query($sql);
     }
+
+    /**
+     * After a pre-auth pending order has existed for 15 minutes or more, we remove it from the system.
+     * At this point, Bolt should have called the "failed_payment" webhook to do this. We are catching
+     * the anomaly cases where this mechanism fails.
+     */
+    public function cleanupOrders() {
+        $expiration_time = Mage::getSingleton('core/date')->gmtDate('Y-m-d H:i:s', time()-(60*15));  // Magento uses GMT to save in DB
```

Comment:
let's move this 15 to constant. Also I suggest we move this to somthing more generous like 6 hours.  we can still run cron every 15min but let's start with less aggressive with deleting data

Reply:
sounds good.

Reply:
@daisy1754 I thought about this a little more.  I accidentally set this at 10 hours instead of 6, and when going to change it, I believe that 6 hours is way too long.  The original 15 minute idea comes from the fact that we are holding up inventory... and in this case, holding inventory on an order that is likely never to succeed.  I believe a much more reasonable time would be 30 minutes.

What do you think?

Reply:
I agree with Leon here - lets start with 30 minutes and we can monitor to tweak it?